### PR TITLE
Bump version to 3.0.0a2

### PR DESCRIPTION
**Pull request by ischoegl**
**Changes proposed in this pull request**

After the merge of #1333 (i.e. CTI/XML are completely removed and remaining `Reaction` specializations are deprecated, plus `yaml2ck` is merged), it may be time for updating the alpha version.

**Checklist**

- [x] The pull request includes a clear description of this code change
- [x] Commit messages have short titles and reference relevant issues
- [x] Build passes (`scons build` & `scons test`) and unit tests address code coverage
- [x] Style & formatting of contributed code follows [contributing guidelines](https://github.com/Cantera/cantera/blob/main/CONTRIBUTING.md)
- [x] The pull request is ready for review

### CODE REVIEW

**Review by speth — CHANGES_REQUESTED** [review]
Agreed, it's about time for this, what with almost 200 commits since v2.6.0. 

However, you forgot about the version number in `README.rst`.

**Comment by ischoegl** [comment]
@speth ... thanks for noticing! This should probably wait until #1338 is merged to include the updated third-body detection logic.

**Review by speth — APPROVED** [review]

**Pull request merged**
